### DISCUSSION TITLE
Where is your favorite city/place of your home prefecture?

### DISCUSSION THREAD

**Discussion by LIFsnn** [NAIST-SE/AttractingContributors#1174] [2024 Attraction Contributions]
Tell me
https://github.com/LIFsnn/Your-favorite-place-of-your-hometown/tree/main

**Comment by wangzaixiaoliunai** [comment]
I have created a PR.
Could you please contribute to mine?
https://github.com/wangzaixiaoliunai/AttractingContributors-musical_instruments.git

**Comment by Marsall-k** [comment]
I have just made a PR on your Repo.
could you contribute to mine?
https://github.com/Marsall-k/AttractingContributors-Pokemon

**Comment by Idescrew** [comment]
I have just made a PR on your Repo.
could you contribute to mine?

https://github.com/Idescrew/AttractingContributors

**Comment by H-S-0811** [comment]
I have just made a PR on your Repo.
could you contribute to mine?

https://github.com/H-S-0811/AttractingContributors
